src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "cognitiveservices/data-plane/Personalizer" -AutoRestVersion "latest" -SdkGenerationDirectory "$PSScriptRoot\Generated\Personalizer" -ConfigFileTag "release_1_0_preview"
```

Comment:
Please generate the code using this script once the Rest spec is merged. This should create a SdkInfo_*.cs and .txt files, please commit these files

Reply:
added